Frontend/src/data/rental/inputs.js

```diff
@@ -293,6 +305,7 @@ export default {
       hidden: (context) => context.createNew,
       component: TextInput,
       props: {
+        quickset:{0:'0'},
```

Comment:
I think that's what you want:
```suggestion
        quickset: (context) => ({ [context.doc.deposit]: context.doc.deposit }),
```

Reply:
perfect! thanks :)